pyflow/host.py

```diff
@@ -958,6 +983,11 @@ def status_cmd(self):
         """*str*: The **ecflow** status command."""
         return self.troika_command("monitor") + " {} %ECF_JOB%".format(self.hostname)
 
+    @property
+    def check_cmd(self):
+        """*str*: The **ecflow** check command."""
+        return self.troika_command("check") + " {} %ECF_JOB%".format(self.hostname)
```

Comment:
This command doesn't exist (yet?) in Troika

Reply:
@oiffrig, I'm not even sure why we need that... we realised it was there for the other hosts. Do you want me to remove it in the meantime?